### PR TITLE
User experience

### PR DESCRIPTION
**Pull request by arcticfloyd1984**
*[GitHub keywords to close any associated issues](https://blog.github.com/2013-05-14-closing-issues-via-pull-requests/)*

## Motivation

*Why should we merge these changes.  If using GitHub keywords to close [issues](https://github.com/poanetwork/blockscout/issues), this is optional as the motivation can be read on the issue page.*

### Enhancements
*Things you added that don't break anything.  Regression tests for Bug Fixes count as Enhancements.*

### Bug Fixes
*Things you changed that fix bugs.  If a fixes a bug, but in so doing adds a new requirement, removes code, or requires a database reset and reindex, the breaking part of the change should be added to Incompatible Changes below also.*

### Incompatible Changes
*Things you broke while doing Enhancements and Bug Fixes.  Breaking changes include (1) adding new requirements and (2) removing code.  Renaming counts as (2) because a rename is a removal followed by an add.*

## Upgrading

*If you have any Incompatible Changes in the above Changelog, outline how users of prior versions can upgrade once this PR lands or when reviewers are testing locally.  A common upgrading step is "Database reset and re-index required".*

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/controllers/html_controller.ex [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-            conn,
-            "index.html",
-            current_path: current_path(conn),
-        )
+    alias BlockScoutWeb.Call
+    alias BlockScoutWeb.Controllers
+    
+
+
+    def show(conn, %{"id" => id}) do
```
Comment (gmjacob) [inline apps/block_scout_web/lib/block_scout_web/controllers/html_controller.ex:9]: is this necessary?

File: apps/block_scout_web/lib/block_scout_web/controllers/transaction_internal_transaction_controller.ex [hunk 2]
```diff
@@ -33,20 +33,151 @@ defmodule BlockScoutWeb.TransactionInternalTransactionController do
           paging_options(params)
         )
 
-      internal_transactions_plus_one = Chain.transaction_to_internal_transactions(transaction, full_options)
+      # internal_transactions_plus_one = Chain.transaction_to_internal_transactions(transaction, full_options)
```
Comment (gmjacob) [inline apps/block_scout_web/lib/block_scout_web/controllers/transaction_internal_transaction_controller.ex:36]: you can uncomment this check for the PR

**Review by gmjacob — CHANGES_REQUESTED** [review]

File: apps/block_scout_web/lib/block_scout_web/controllers/html_controller.ex (hunk 1 above)
Comment (arcticfloyd1984) [reply apps/block_scout_web/lib/block_scout_web/controllers/html_controller.ex:9]: html_controller is only for testing purposes. I will remove this when everything is done, the functionality is added in the transaction_internal_transaction_controller.ex

File: apps/block_scout_web/lib/block_scout_web/controllers/transaction_internal_transaction_controller.ex (hunk 2 above)
Comment (arcticfloyd1984) [reply apps/block_scout_web/lib/block_scout_web/controllers/transaction_internal_transaction_controller.ex:36]: Ok

**Pull request merged**